pkgs/development/python-modules/notmuch2/default.nix

```diff
@@ -13,7 +14,12 @@ buildPythonPackage {
 
   sourceRoot = "notmuch-${notmuch.version}/bindings/python-cffi";
 
-  buildInputs = [ python notmuch cffi ];
+  nativeBuildInputs = [
+    python3Packages.cffi
```

Comment:
can't it be cffi ? or python.pkgs.cffi

Reply:
26f5078521d07579a6b76a52dd04d6f0dda50567